### DISCUSSION TITLE
Host over LAN

### DISCUSSION THREAD

**Discussion by Sir-Skunk-a-lot** [soda3x/ArmaReforgerServerTool#83] [Q&A]
Is it possible to host the server over LAN like in the game?

**Comment by soda3x** [comment]
Not possible with this tool

**Reply by Sir-Skunk-a-lot** [reply]
Thanks for the quick answer. Any plan for it of not?

**Reply by soda3x** [reply]
If its possible, it's a possibility that I will get to it. Currently its not the focus of the tool though as theres many other features required first.
